neuralprophet/plot_model_parameters.py

```diff
@@ -26,7 +26,33 @@
     log.error("Importing matplotlib failed. Plotting will not work.")
 
 
-def plot_parameters(m, forecast_in_focus=None, weekly_start=0, yearly_start=0, figsize=None):
+def check_df_name(m, df_name):
+    """Check if name of dataframe provided for referring to local modeling params is valid
+    Args:
+        m (NeuralProphet): fitted model.
+        df_name: name of dataframe to refer to data params from original list of train dataframes (used for local normalization in global modeling)
+
+    """
+    if not m.local_modeling and df_name is not None:
+        log.warning("Global modeling local normalization was not used - ignoring given df_name")
+    if m.local_modeling:
+        if df_name is None:
+            raise ValueError(
+                "Global modeling local normalization was used. Please insert name of dataframe to refer to in order to perform desired plot"
+            )
+        if not isinstance(df_name, str):
+            raise ValueError(
+                "Global modeling local normalization was used. Please insert a string with the name of single dataframe to refer to in order to perform desired plot"
+            )
+        if df_name not in m.data_params.df_names:
+            raise ValueError(
+                "Global modeling local normalization was used. Name {name!r} missing from data params".format(
+                    name=df_name
+                )
+            )
+
+
+def plot_parameters(m, forecast_in_focus=None, weekly_start=0, yearly_start=0, figsize=None, df_name=None):
```

Comment:
what happens if df_name is none? are the plots just in normalized scale?

Reply:
if df_name is none and local_modeling is true we raise an error (we can't plot the parameters unless we provide a name to refer to the data_params).

Reply:
I think that for plotting we could allow this - we would just have to print an info/warning statement that plots are in normalized scale.

Reply:
Ok! Makes sense, would we still have the option to provide a name referring to data params?

Reply:
@ourownstory I don't believe this is feasible. We can discuss this tomorrow.

Reply:
We can't predict the trend with no data_params related to df.